docs/content/latest/benchmark/tpcc-ysql/10000-wh/tpcc-ysql.md

```diff
@@ -1,19 +1,17 @@
-### TPC-C Load Phase
+## TPC-C Load Phase
 
-Before starting the workload, you will need to load the data first. Make sure
-to replace the IP addresses with that of the nodes in the cluster.
+Before starting the workload, you need to load the data. In addition, you need to ensure that you exported a list of all IP addresses of all the nodes involved.
 
-For 10k warehouses, we would need 10 clients of type c5.2xlarge to drive the benchmark.
-Since there are multiple clients we perform 3 steps to load the data
+For 10k warehouses, you would need ten clients of type c5.2xlarge to drive the benchmark.
+For multiple clients, you need to perform three steps.
 
-First we create the database and the corresponding tables. The following command is to be executed from only one client.
-Make sure that you export the list of all IP addresses of all the nodes involved before executing.
+First, you create the database and the corresponding tables. The following command is to be executed from only one client:
 
```

Comment:
The following command is to be executed from only one client: -> Execute the following command from one of the clients:

Reply:
Changed